modules/qna/src/views/full/Components/QnA.tsx

```diff
@@ -219,20 +219,20 @@ const QnA: FC<Props> = props => {
       </div>
       {expanded && (
         <div key={contentLang} className={style.collapsibleWrapper}>
-          {!isLite && (
+          {/* {!isLite && (
             <ContextSelector
               className={cx(style.contextSelector)}
-              contexts={data.contexts}
+              contexts={data.topicName}
               customIdSuffix={id}
               saveContexts={contexts =>
                 updateQnA({
                   id,
-                  data: { ...data, contexts }
+                  data: { ...data, topicName }
                 })
               }
               bp={bp}
             />
-          )}
+          )} */}
```

Comment:
Commented, should it be removed?

Reply:
I guess so, it was old word from maxime and we don't use context anymore....